code/modules/admin/admin.dm

```diff
@@ -316,22 +316,21 @@ GLOBAL_VAR_INIT(nologevent, 0)
 	if(!check_rights(R_ADMIN))
 		return
 
-	var/dat = {"
-		<center><B>Game Panel</B></center><hr>\n
-		<A href='?src=[UID()];c_mode=1'>Change Game Mode</A><br>
-		"}
+	var/dat = "<center>"
```

Comment:
Can you mimic the changes in https://github.com/ParadiseSS13/Paradise/pull/20034 in here?

The list.Join() vs string concat change is huge

Reply:
You resolved this without actually doing the change

Reply:
@S34NW  do this please

Reply:
turns out I did it for the ban panel not this without realising. 10/10